apps/sveltekit-app/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "sveltekit-app",
+  "version": "0.0.1",
+  "private": true,
+  "scripts": {
+    "dev": "vite dev",
+    "build": "vite build",
+    "preview": "vite preview",
+    "test": "playwright test",
+    "check": "svelte-kit sync && svelte-check --tsconfig ./tsconfig.json",
```

Comment:
should these be moved to tasks?

Reply:
Oops, I've created the tasks but missed removing the scripts